week_1_basics_n_setup/homework.md

```diff
@@ -1,3 +1,54 @@
 ## Week 1 Homework
 
-Coming soon
\ No newline at end of file
+In this homework we'll prepare the environment 
+and practice with terraform and SQL
+
+## Question 1. Google Cloud SDK
+
+Install Google Cloud SDK. What's the version you have? 
+
+To get the version, run `gcloud --version`
+
+## Question 2. Terraform 
+
+Install Terraform. What's the version you have?
+
+Run `terraform -version` to get it
+
+
+## Prepare Postgres 
+
+Run Postgres and load data as shown in the videos
+
+We'll use the yellow taxi trips from January 2021:
+
+```bash
+wget https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv
+```
+
+Download this data and put it to Postgres
+
+## Question 3. Count records 
+
+How many taxi trips were there on January 15?
+
+## Question 4. Average
+
+Find the largest tip for each day. 
```

Comment:
I think here  and the next line you meant trip

Reply:
Actually I did not!

Reply:
I was referring to `tip_amount` - made it clear that it's not a typo. Thanks!